Description: Provide version#"git description".
 Since we aren't necessarily building the package from a git
 repository (and even if we are, we're likely building from a Debian
 packaging branch and not an upstream one), we hardcode the git
 description ahead of time in this patch.  Refresh before every
 release by running 'debian/rules update-git-description'.
Author: Doug Torrance <dtorrance@piedmont.edu>
Forwarded: not-needed
Last-Update: 2020-12-16

--- a/M2/configure.ac
+++ b/M2/configure.ac
@@ -562,7 +562,7 @@
 NEW_COMMITS=`git rev-list $(git rev-list -1 HEAD $srcdir/VERSION)..HEAD --count`
 GIT_COMMIT=`git describe --dirty --always --match HEAD`
 if test "$NEW_COMMITS-$GIT_COMMIT" = "-"
-then GIT_DESCRIPTION=version-$PACKAGE_VERSION-unknown
+then GIT_DESCRIPTION=version-1.16.99-77-4077f1b1b
 else GIT_DESCRIPTION=version-$PACKAGE_VERSION-$NEW_COMMITS-$GIT_COMMIT
 fi
 AC_DEFINE_UNQUOTED(GIT_DESCRIPTION,"$GIT_DESCRIPTION",[summary of git status])
